R/sapflow-Rs.Rmd

```diff
@@ -518,7 +542,7 @@ bind_rows(df1, df2, df3, df4, df5, df6) %>%
 
 
 ```{r sc-compute, echo=FALSE, message=FALSE, warning=FALSE}
-fsd(sunny_days, climate_data, lookahead = 7, constraints) %>% 
+fsd(sunny_days, climate_data, lookahead = 14, constraints) %>% 
```

Comment:
@stephpenn1 note this change - didn't mean to include it here. Can revert if you want.